packages/fxa-content-server/server/lib/sentry.js

```diff
@@ -91,6 +91,17 @@ function tryCaptureValidationError(err) {
     }
 
     if (errorDetails) {
+      // We are ignoring all navigation timings issues until we come up with a better
+      // solution for capturing timing data. See FXA-7005.
+      if (
+        typeof errorDetails.details.some === 'function' &&
```

Comment:
nit: should we use something like [`Array#isArray()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray) to see if `errorDetails.details` is an Array (vs property sniffing using `typeof`)?
Unless this was specifically checking for that fn/polyfill, although per https://caniuse.com/mdn-javascript_builtins_array_some it looks like it already has pretty wide support.